com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/SyncFlowableStep.java

```diff
@@ -113,7 +118,18 @@ protected void onStepError(DelegateExecution context, Exception e) throws Except
 
     protected void processException(Exception e, String detailedMessage) throws Exception {
         e = handleControllerException(e);
-        throw getExceptionConstructor(e).apply(e, detailedMessage + ": " + e.getMessage());
+        String tail = computeTail(e);
+        throw getExceptionConstructor(e).apply(e, detailedMessage + ": " + e.getMessage() + " " + tail);
+    }
+
+    protected String computeTail(Exception e) {
+        if(e instanceof CloudServiceBrokerException) {
```

Comment:
Maybe we should avoid adding anything for ContentException.
Imagine we have invalid mtad.yaml, e.g. some required field is missing. It is clear the it is modelling error, and incident could not help much.